langserver/langserver.py

```diff
@@ -335,7 +335,7 @@ def __init__(self, code, message, data=None):
         self.data = data
 
 
-class ThreadingTCPServer(socketserver.ThreadingMixIn, socketserver.TCPServer):
+class ThreadingTCPServer(socketserver.ForkingMixIn, socketserver.TCPServer):
```

Comment:
probably should update the name of this class as well. But yeah, I think it is as easy as this :)

I would test it locally by having more than one repo open in sourcegraph